atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ExpiringKeyValueService.java

```diff
@@ -27,23 +27,39 @@
  *
  * Contract with the user:
  * - This is for space and deletion time savings only.
- * - This does not fulfill the same goals/legal requirements as hard delete for sensitive data. (though data written by this service can itself be deleted by normal hard delete routines)
- * - // todo Only tables marked with 'AllowsSelfExpiringData' in their schema may use these features
+ * - This does not fulfill the same goals/legal requirements as hard delete for sensitive data.
+ *   (though data written by this service can itself be deleted by normal hard delete routines)
+ * - // TODO: Only tables marked with 'AllowsSelfExpiringData' in their schema may use these features
  *
- * - Normal atlasdb guarantees may be broken. Data written with expiration times are:
+ * - Normal AtlasDB guarantees may be broken. Data written with expiration times are:
```

Comment:
Nope, I use `F2` to go through all of the checkstyle errors but when they run out it gives me random warnings and spelling mistakes and sometimes I fixed some of those as well!